examples/plugins/windows-plugin/plugin.yaml

```diff
@@ -0,0 +1,28 @@
+podSpec:
+  nodeSelector:
```

Comment:
Will you be including a README or anything along with this to explain what is required to create a Windows plugin or will that be included as separate docs?

Reply:
Yes I'll add a readme to this plugin folder and talk about windows specific points of interest. Seems like a reasonable place to put it; may add a link elsewhere in docs to point to this as well.